hera_cal/lstbin.py

```diff
@@ -650,13 +652,12 @@ def lst_bin_files(data_files, input_cals=None, dlst=None, verbose=True, ntimes_p
 
                     # load data: only times needed for this output LST-bin file
                     hd = io.HERAData(data_files[j][k], filetype='uvh5')
-                    try:
-                        data, flags, nsamps = hd.read(bls=blgroup, times=tarr[tinds])
```

Comment:
okay so to see if I have this right: this line can fail when only one (or more) baselines in `blgroup` are not present in the `hd` object, even if there are other baselines in `blgroup` that do exist. if that's right, then I think this is indeed a bug. thanks for catching

Reply:
Yes, that's right. It fails if you ask for a baseline with an antenna not in the data.